### PR TITLE
Group B - Steve Bloniarz

### PR DESCRIPTION
**Pull request by stevebloniarz**

### CODE REVIEW

File: _posts/2020-06-05-steve.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: A place I'd like to visit when things get better...
+author: your name here
+date: 2020-06-05
+---
+
+A place I'd like to visit when things get better = Breweries
+
+![CT Valley Brewing](https://media-cdn.tripadvisor.com/media/photo-s/13/a9/ee/a4/20180711-163936-largejpg.jpg)
+
+Comment your reply here.
```
Comment (alliyahtaylor) [inline _posts/2020-06-05-steve.md]:
```suggestion
Skittles replied: Let's do a group outing when the world is normal again!
```

Comment (ServiceNowTestAccount) [inline _posts/2020-06-05-steve.md]:
```suggestion
nice choice - Richie
```

Comment (hunthenriksen) [inline _posts/2020-06-05-steve.md]:
```Hunter replied
Bottoms up.
```

**Comment by zach004** [comment]
Cheers

**Comment by asingh9** [comment]
beautiful

File: _posts/2020-06-05-steve.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: A place I'd like to visit when things get better...
+author: your name here
```
Comment (asingh9) [inline _posts/2020-06-05-steve.md]: @stevebloniarz put your name!

Comment (stevebloniarz) [inline _posts/2020-06-05-steve.md]:
```suggestion
author: Steve Bloniarz
```

Comment (stevebloniarz) [reply _posts/2020-06-05-steve.md]: whoops, done!

**Pull request merged**